src/status_im/contexts/communities/actions/addresses_for_permissions/view.cljs

```diff
@@ -32,22 +31,24 @@
 
       (= type constants/community-token-type-erc721)
       (assoc :collectible-name    name
-             :collectible-img-src (resources/get-mock-image :collectible))
+             :collectible-img-src (token-images-map sym))
 
       (= type constants/community-token-type-erc20)
-      (assoc :amount (str (money/token->unit amount decimals))
-             :token  (:symbol balance)))))
+      (assoc :amount        (str (money/token->unit amount decimals))
+             :token         (:symbol balance)
+             :token-img-src (token-images-map sym)))))
 
 (defn- account-item
   [{:keys [color address name emoji]} _ _
    {:keys [selected-addresses community-id share-all-addresses? community-color]}]
-  (let [balances (rf/sub [:communities/permissioned-balances-by-address community-id address])]
+  (let [balances         (rf/sub [:communities/permissioned-balances-by-address community-id address])
+        token-images-map (rf/sub [:communities/token-requirements-images community-id])]
```

Comment:
A suggestion I have is to rename the sub to `:communities/token-images-by-symbol`, and rename the binding to `images-by-symbol` for example. Suffixing the type of the data structure (`map` in this case) is unconventional in Clojure. The convention `x-b-y` is powerful because it tells us immediately what the map is indexed by.